packages/vuepress-theme-odyssey/global-components/Visual.vue

```diff
@@ -27,35 +24,39 @@ export default {
   props: {
     variant: {
       type: String,
-      required: false
+      required: false,
+      default: null
     },
     layout: {
       type: String,
-      required: false
+      required: false,
+      default: null
     },
     theme: {
       type: String,
-      required: false
+      required: false,
+      default: null
     },
     content: {
       type: String,
-      required: false
+      required: false,
+      default: null
     },
     fade: {
       type: Boolean,
       required: false
     }
   },
-  data: function () {
+  data: function() {
     return {
       labels: {
-        'positive': 'Do',
-        'negative': `Don't`,
-        'smiley-positive': 'Best!',
-        'smiley-neutral': 'Good',
-        'smiley-negative': 'Bad'
+        positive: "Do",
+        negative: `Don't`,
```

Comment:
nit: This can be double quotes

```suggestion
        negative: "Don't",
```